src/platform/user/authentication/components/SignInModal.jsx

```diff
@@ -140,7 +140,7 @@ class SignInModal extends React.Component {
           </div>
           <div className="columns usa-width-one-half medium-6">
```

Comment:
Can we remove the columns and medium-6 classes here, too?

Reply:
Sure